README.md

```diff
@@ -180,6 +180,47 @@ Providers
   </tr>
 </table>
 
+Example
+-------
+To generate a certificate for an apache2 website you can use code like this:
+
+    # Set up contact information. Note the mailto: notation
+    node.set['letsencrypt']['contact'] = [ 'mailto:me@example.com' ] 
+    # Real certificates please...
+    node.set['letsencrypt']['endpoint'] = 'https://acme-v01.api.letsencrypt.org' 
+
+    site="example.com"
+    sans=Array[ "www.#{site}" ]
+
+    # Set up your server here...
```

Comment:
Setup of the webserver should happen after calling the `selfsigned` but before the `certificate` provider